src/templates/common/link/index.test.tsx

```diff
@@ -1,38 +0,0 @@
-import { render } from '@testing-library/react'
```

Comment:
This is not used; all instances of <Link> are imported directly from 'next/link'